File: Python/List-2/centered_average.py
def centered_average(nums):
  
  count = 0
  min = nums[0]
  max = nums[0]
  
  for i in range(len(nums)):
    if (min > nums[i]): min = nums[i]
    if (max < nums[i]): max = nums[i]
    count += nums[i]
  
  return (count - max - min)/(len(nums) - 2)

File: Java/Array-2/centeredAverage.java
public int centeredAverage(int[] nums) {
  int len = nums.length, max = nums[0], min = nums[0], sum = 0;
  for (int i = 0; i < len; i++) {
    max = Math.max(nums[i], max);
    min = Math.min(nums[i], min);
    sum += nums[i];
  }
  return (sum - max - min)/(len - 2);
}
